docs/src/content/docs/index.mdx

```diff
@@ -4,114 +4,137 @@ description: Terrateam documentation
+        <LinkCard title="Installation" href="/quickstart-guide" />
 	</Card>
 	<Card title="GitOps Workflow" icon="rocket">
-		Terraform operations are triggered by pull requests and comments, making it easy to review and approve changes before they are applied.
+		Manage infrastructure via pull requests with automated plans and controlled applies.
+        <LinkCard title="Workflows" href="/getting-started/plan-and-apply" />
 	</Card>
-	<Card title="Secure Authentication" icon="sun">
-		Terrateam authenticates with your cloud provider using OIDC, which enhances security and reduces the exposure of user credentials.
+	<Card title="Enterprise Security" icon="sun">
+		OIDC authentication, access controls, and policy enforcement.
+        <LinkCard title="Security" href="/security-and-compliance" />
 	</Card>
-	<Card title="Locks and Concurrency" icon="seti:lock">
-		When multiple team members are working on the same Terraform project, locks play a crucial role in coordinating their efforts and preventing conflicting changes.
+	<Card title="Resource Coordination" icon="seti:lock">
+		Smart locking prevents conflicts when multiple team members work simultaneously.
+        <LinkCard title="Locks" href="/advanced-workflows/locks-and-concurrency" />
 	</Card>
-	<Card title="Role-Based Access Control" icon="error">
-		With Terrateam, you can define fine-grained access control policies to manage who can perform specific operations on your infrastructure.
+	<Card title="Access Control" icon="error">
+		Fine-grained policies determine who can perform specific infrastructure operations.
+        <LinkCard title="RBAC" href="/security-and-compliance/role-based-access-control" />
 	</Card>
 	<Card title="Custom Workflows" icon="puzzle">
-		Terrateam enables custom Terraform workflows, such as automating Slack notifications post-apply, tailored to your team's needs.
+		Automate processes like post-apply notifications tailored to your team's needs.
+        <LinkCard title="Custom Workflows" href="/configuration-reference/workflows" />
 	</Card>
-	<Card title="Multiple Environments" icon="star">
-		Easily manage multiple environments (e.g., development, staging, production) and multiple cloud providers with Terrateam's powerful configuration options.
+	<Card title="Multi-Environment" icon="star">
+		Consistently manage development, staging, and production environments.
+        <LinkCard title="Environments" href="/advanced-workflows/multiple-environments" />
 	</Card>
 	<Card title="Policy Enforcement" icon="document">
-		Integrate policy testing with OPA Conftest to ensure your infrastructure changes adhere to best practices and compliance requirements.
+		Integrate OPA Conftest to ensure compliance with best practices.
+        <LinkCard title="OPA Integration" href="/security-and-compliance/policy-enforcement-with-opa" />
 	</Card>
 </CardGrid>
 
-## Terraform the GitOps Way
+## How Terrateam Works
```

Comment:
Before outlining the steps, include a one-sentence summary describing the process's purpose. This can help establish expectations before delving into the specifics.